.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bugs
+about: Found a bug in OpenTTD?
+title: "Bug Report"
+---
+
+## Version of OpenTTD
+<!-- Indicate what version of OpenTTD you are using, including your OS. -->
```

Comment:
I see I'm late to this (already merged), but I'd really suggest changing to `fill in below` because I see too many who replace the heading or similar for these.

Reply:
I was hoping this would already help, but if you can PR that, that sounds like a good addition :)

It is never too late to improve on each others work ;)